.ide-toolbox.metadata.json

```diff
@@ -0,0 +1,199 @@
+                }
+            ]
+        }
+    ],
+    "providers": [
+        {
+            "name": "contao_system_classes",
+            "defaults": {
+                "icon": "com.jetbrains.php.PhpIcons.CLASS"
+            },
+            "source": {
+                "contributor": "sub_classes",
+                "parameter": "Contao\\System"
+            }
+        },
+        {
+            "name": "contao_model_classes",
+            "defaults": {
+                "icon": "com.jetbrains.php.PhpIcons.CLASS"
+            },
+            "source": {
+                "contributor": "sub_classes",
+                "parameter": "Contao\\Model"
+            }
+        },
+        {
+            "name": "contao_framework_classes",
+            "defaults": {
+                "icon": "com.jetbrains.php.PhpIcons.CLASS"
+            },
+            "items": [
+                {
+                    "lookup_string": "Contao\\System",
+                    "type": "Contao\\System"
+                }, {
+                    "lookup_string": "Contao\\Config",
+                    "type": "Contao\\Config"
+                }
+            ]
+        },
+        {
+            "name": "contao_datacontainers",
+            "lookup_strings": [
+                "tl_article",
+                "tl_content",
+                "tl_cron",
+                "tl_files",
+                "tl_form",
+                "tl_form_field",
+                "tl_image_size",
+                "tl_image_size_item",
+                "tl_layout",
+                "tl_log",
+                "tl_member",
+                "tl_member_group",
+                "tl_module",
+                "tl_page",
+                "tl_settings",
+                "tl_style",
+                "tl_style_sheet",
+                "tl_templates",
+                "tl_theme",
+                "tl_undo",
+                "tl_user",
+                "tl_user_group"
+            ]
+        },
+        {
+            "name": "contao_languagefiles",
+            "lookup_strings": ["default", "explain"]
+        },
+        {
+            "name": "contao_model_options",
+            "items": [
+                {
+                    "lookup_string": "column",
+                    "type_text": "string|array"
+                },
+                {
+                    "lookup_string": "value",
+                    "type_text": "string|array"
+                },
+                {
+                    "lookup_string": "sorting",
+                    "type_text": "SQL"
+                },
+                {
+                    "lookup_string": "limit",
+                    "type_text": "integer"
+                },
+                {
+                    "lookup_string": "offset",
+                    "type_text": "integer"
+                },
+                {
+                    "lookup_string": "return",
+                    "type_text": "Model|Collection|Array"
+                }
```

Comment:
Here are the options I have found:

 * [ ] `table`
 * [x] `column`
 * [x] `value`
 * [x] `limit`
 * [x] `offset`
 * [x] `return` 
 * [ ] `order` (probably what you mean by `sorting`)